src/Compilers/Core/Portable/InternalUtilities/IsExternalInit.cs

```diff
@@ -21,4 +21,12 @@ internal static class IsExternalInit
     }
 }
 
+#else
+
+using System.Runtime.CompilerServices;
+
+#pragma warning disable RS0016 // Add public types and members to the declared API (this is a supporting fowarder for an internal polyfill API)
```

Comment:
>fowarder [](http://example.com/codeflow?start=105&length=8)

Typo?

Reply:
Was going to fix this but backlogged on another work item and didn't want to reset a green CI.